repo/packages/B/bookkeeper/0/marathon.json.mustache

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "{{bookkeeper.app_id}}",
+  "cpus":{{bookkeeper.cpus}},
+  "mem": {{bookkeeper.mem}},
+  "instances": {{bookkeeper.instances}},
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/bk/journal",
+        "hostPath": "/mnt/journal",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/bk/index",
+        "hostPath": "/mnt/index",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/bk/ledgers",
+        "hostPath": "/mnt/ledgers",
+        "mode": "RW"
+      },
+  
+      {
+        "containerPath": "persist",
+        "mode": "RW",
+        "persistent": {
+          "size": 70
+        }
+      }
+    ],
+
+    "docker": {
+      "image": "{{resource.assets.container.docker.bookkeeper}}",
+      "network": "HOST",
+      "portMappings": [
+        { "containerPort": 3181, "hostPort": 3181, "protocol": "tcp" },
```

Comment:
probably want hostPort (for these two lines) to be configurable with a sensible default. Only hard-code the default host port to non-zero if you really need it that way, otherwise please default to zero.